dotCMS/src/main/java/com/dotmarketing/beans/Host.java

```diff
@@ -109,7 +109,7 @@ public void setHostThumbnail(File thumbnailInode) {
 	}
 
 	public boolean isDefault() {
-		return (Boolean) map.get(IS_DEFAULT_KEY);
+		return  map.containsKey(IS_DEFAULT_KEY)? (Boolean) map.get(IS_DEFAULT_KEY):Boolean.FALSE;
 	}
```

Comment:
Map.getOrDefault would suit much better here.  I think

Reply:
I agree you can replace by
```
map.getOrDefault (IS_DEFAULT_KEY, false);
```

Reply:
Done